app.json

```diff
@@ -53,11 +53,6 @@
         "windows": [
             {
                 "layout": {
-                    "settings": {
```

Comment:
should we keep these to make sure we find any bugs?

Reply:
they're in types now, should document them somewhere too

Reply:
yup